lib/vow.js

```diff
@@ -348,12 +348,12 @@ Promise.prototype = /** @lends Promise.prototype */ {
     },
 
     /**
-     * Adds reactions to promise.
+     * Adds reactions to the promise.
      *
-     * @param {Function} [onFulfilled] Callback that will to be invoked with the value after promise has been fulfilled
-     * @param {Function} [onRejected] Callback that will to be invoked with the reason after promise has been rejected
-     * @param {Function} [onProgress] Callback that will to be invoked with the value after promise has been notified
-     * @param {Object} [ctx] Context of callbacks execution
+     * @param {Function} [onFulfilled] Callback that is going to be invoked with a provided value after the promise has been fulfilled
+     * @param {Function} [onRejected] Callback that is going to be invoked with a provided reason after the promise has been rejected
+     * @param {Function} [onProgress] Callback that is going to be invoked with a provided value after the promise has been notified
```

Comment:
I'm pretty sure "to be going to" can be used here - [wiki page on "going-to"](https://en.wikipedia.org/wiki/Going-to_future#Usage).
It's just more informal. It can be replaced with the more formal:
```
Callback that will be invoked...
```